### DISCUSSION TITLE
efmnhtransactions - Transactions Module

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#894] [Structural Changes]
This is a discussion to collate and discuss changes that we would like to see in the transactions module in EMu. These may be structural or data.

[OPEN STRUCTURAL CHANGE ISSUES](https://github.com/fieldmuseum/EMu-Documentation/issues?q=is%3Aissue+is%3Aopen+label%3A%22structural+changes%22+label%3Aefmnhtransactions)

**Comment by rondlg** [comment]
Now that Photo Archives accession records are being added to EMu we need a new Accession Number prefix: P

**Reply by rondlg** [reply]
Change request submitted for quote: https://github.com/fieldmuseum/EMu-Documentation/wiki/efmnhtransactions-module#reasons-for-change-request-version-5

**Comment by cmaves3** [comment]
Date Accessioned (AccDateAccessioned) in Transactions is a Text field that should be changed to a Date field

**Reply by rondlg** [reply]
Change request submitted for quote: https://github.com/fieldmuseum/EMu-Documentation/wiki/efmnhtransactions-module#reasons-for-change-request-version-5

**Comment by rondlg** [comment]
Deaccession Tool - https://github.com/fieldmuseum/EMu-Documentation/issues/1118

- Also referenced in ecatalogue

**Comment by rondlg** [comment]
Custody hierarchy: https://github.com/fieldmuseum/EMu-Documentation/issues/1122

**Comment by rondlg** [comment]
Credit To summarystring update https://github.com/fieldmuseum/EMu-Documentation/discussions/1286

**Comment by mariegeorg** [comment]
Please include a reverse attachment field so one can see all the catalog records attached to a given Accessions/Acquisitions-type efmnhtransaction record.

**Comment by rondlg** [comment]
Exhibitions acquisition tab 
- https://github.com/fieldmuseum/EMu-Documentation/issues/1087
- https://github.com/fieldmuseum/EMu-Documentation/issues/1088

**Comment by rondlg** [comment]
Transaction tab consolidation: https://github.com/fieldmuseum/EMu-Documentation/issues/1095

**Comment by rondlg** [comment]
updates to details tab: https://github.com/fieldmuseum/EMu-Documentation/issues/1096

**Comment by rondlg** [comment]
updates to movements tab https://github.com/fieldmuseum/EMu-Documentation/issues/1097

**Comment by rondlg** [comment]
Field labels: https://github.com/fieldmuseum/EMu-Documentation/issues/1102

**Comment by rondlg** [comment]
Date authorised : https://github.com/fieldmuseum/EMu-Documentation/issues/1104

**Comment by rondlg** [comment]
Remarks and tab consolidation: https://github.com/fieldmuseum/EMu-Documentation/issues/1107

**Comment by rondlg** [comment]
Deaccessions tab: https://github.com/fieldmuseum/EMu-Documentation/issues/1109

**Comment by rondlg** [comment]
Rights connection and read-thru information: https://github.com/fieldmuseum/EMu-Documentation/issues/1112

**Comment by rondlg** [comment]
Making for use by multivalued: https://github.com/fieldmuseum/EMu-Documentation/issues/1116

**Comment by rondlg** [comment]
Deaccession tool request: https://github.com/fieldmuseum/EMu-Documentation/issues/1118

**Comment by rondlg** [comment]
Preps/Part lookup list consoludation: https://github.com/fieldmuseum/EMu-Documentation/issues/1123

**Comment by rondlg** [comment]
Replace Geography text field with attachment to esites: https://github.com/fieldmuseum/EMu-Documentation/issues/1124

**Comment by fmjjones** [comment]
One Objects and Uncat Objects tab: https://github.com/fieldmuseum/EMu-Documentation/issues/1092
